scripts/setup-test.sh

```diff
@@ -7,13 +7,18 @@ else
     export GNUPGHOME="$MAILPILE_HOME"
 fi
 
-if [ -d '/usr/local/Cellar/gnupg/1.4.16/bin' ]; then
-    export PATH=/usr/local/Cellar/gnupg/1.4.16/bin:$PATH
+if [ "$VIRTUAL_ENV" -a -e "$VIRTUAL_ENV/bin/gpg2" ]; then
+    GPG_BINARY="$VIRTUAL_ENV/bin/gpg2"
+else
+    if [ -d '/usr/local/Cellar/gnupg/1.4.16/bin' ]; then
```

Comment:
That might need cleaning up at some point, but I'd rather leave it as is for this PR. This was used last time I did a Mac OS X build / test cycle, the next person to actually do some Mac work would be the right person to tweak that.